Backend/app/rider-platform/rider-app/Main/spec/Storage/RecentLocation.yaml

```
@@ -0,0 +1,46 @@
+  LatLong: Kernel.External.Maps.Types
+
+RecentLocation:
+  tableName: recent_location
+
+  types:
+    EntityType:
+      enum: "MULTIMODAL, BUS, METRO, TAXI"
+
+
+  fields:
+    id : Id RecentLocation
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)